### PR TITLE
Admin CRUD kudo

### PR DESCRIPTION
**Pull request by mdobosz3**
Add the ability to CDUD kudos for the admin

### CODE REVIEW

**Review by ciejj — APPROVED** [review]
Looks good! 👍  I have left some suggestions

File: app/views/admin/_adminheader.html.erb [hunk 1]
```diff
@@ -9,6 +9,10 @@
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
 
+    <li class="nav-item">
+      <%= link_to "Kudos index", admin_kudos_path, class: "nav-link" %>
+    </li>
+
     <li class="nav-item">
       <%= link_to "Log Out", destroy_admin_user_session_path, method: :delete, class: "nav-link" %>
     </li>
```
[the remark this answers is not in the document]
Comment (mdobosz3) [reply app/views/admin/_adminheader.html.erb]: In my code, `<li>` is in `<ul>` but it should be shifted further to the right. Now it should be clearer :)

File: app/controllers/admin/kudos_controller.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    # DELETE /admin/kudos/1
+    def destroy
+      kudo.destroy
+      redirect_to admin_kudos_url, notice: 'Kudo was successfully destroyed.'
+    end
+
+    private
+
+    # Use callbacks to share common setup or constraints between actions.
```
[the remark this answers is not in the document]
Comment (mdobosz3) [reply app/controllers/admin/kudos_controller.rb]: I'm using `rails routes` but when executing the command `rails generate scaffold_controller` the comments were added automatically. I already deleted them

File: app/views/admin/kudos/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<h1>New Admin Kudo</h1>
```
[the remark this answers is not in the document]
Comment (mdobosz3) [reply app/views/admin/kudos/new.html.erb]: You have right. I changed this

File: spec/system/admin_kudo_spec.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  let!(:kudo) { build(:kudo) }
+
+  it 'crud kudo' do
+    visit root_path
+    click_link 'Admin Log In'
+    fill_in 'Email', with: admin_user.email
+    fill_in 'Password', with: admin_user.password
+    click_button 'Log in'
+    expect(page).to have_content 'Signed in successfully.'
+    expect(page).to have_content 'Admin Dashboard'
```
[the remark this answers is not in the document]
Comment (mdobosz3) [reply spec/system/admin_kudo_spec.rb]: It is very helpful, I will read more about it :). I added automatic admin login

**Comment by oskarlakner** [comment]
well done Michal 🙌

**Pull request merged**